### ISSUE TITLE
purchase button not working

### ISSUE THREAD

**Issue by sai-sy** [chester-hill-solutions/callcaster#508]
24-25 tanveer shanawaz campaign

![image](https://github.com/user-attachments/assets/46c2d0cb-2355-4d52-87e8-c0453b2ba009)

**Comment by wra-sol** [comment]
It's working for me. Can you refresh and verify it's not just an issue with optimistic updates

**Comment by sai-sy** [comment]
yea I've refreshed and still no purchase. RBAC? But I'm the owner so idk

**Comment by parthiv-2006** [comment]
Purchase button seems to work but there's no real visual indicator that the purchase was made, only thing that happens is the number purchased disappears from the list of numbers

**Comment by sai-sy** [comment]
@parthiv-2006 does the newly purchased number appear in the purchased/my numbers section without a refresh?

**Comment by parthiv-2006** [comment]
@sai-sy nope, you have to refresh the screen, same thing for when you delete an existing number. Doesn't get removed visually till refresh

**Comment by sai-sy** [comment]
@parthiv-2006 make that it's own feature request issue and tag wra-sol. I'm moving this to done
